### PR TITLE
feat: add git-continue

### PR DESCRIPTION
**Pull request by oikarinen**
Also refactor `git-abort` to parse action from the called file name.

There's a promise for this in #865 but that was a long time ago.

### CODE REVIEW

File: bin/git-abort [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
+    echo "No active operation found" >&2
+    exit 1
+  fi
+  if [[ "$(echo "$op" | wc -l)" -gt 1 ]]; then
+    echo "Multiple active operations found: $op" >&2
+    exit 1
+  fi
+}
+
+function discover_action() {
```
Comment (hyperupcall) [inline bin/git-abort:27]: I'm not a huge fan of this function, but I'm good with this for now.

**Review by hyperupcall — CHANGES_REQUESTED** [review]
Thanks! Looks good, I have a few suggestions

File: bin/git-abort [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
 
-gitdir="$(git rev-parse --git-dir)" || exit
-opfound=
-fcnt=
-for i in cherry-pick merge rebase revert; do
-  f=${i^^}
-  f=${f/-/_}
-  test -f "${gitdir}/${f}_HEAD" && fcnt=1$fcnt && opfound=$i
-done
+set -xuo pipefail
```
[the remark this answers is not in the document]
Comment (oikarinen) [reply bin/git-abort:3]: Hey thanks for the catch, looks like copilot had a clitch that i didn't notice 🥹

File: tests/helper.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
-import os, subprocess, shutil, tempfile
+import os
+import subprocess
+import shutil
+import tempfile
 from git import Repo, GitCommandError
-from testpath import MockCommand, modified_env
```
Comment (oikarinen) [inline tests/helper.py:3]: These were not used, sorry for whitespace change for the file that `ruff` caused.

**Comment by hyperupcall** [comment]
Thanks for the changes, LGTM. @spacewander what do you think?

**Review by hyperupcall — APPROVED** [review]

**Comment by spacewander** [comment]
Merged. Thanks!

**Pull request merged**